### PR TITLE
moved to kafka-clients 2.8

### PR DESCRIPTION
**Pull request by jsulmont**
### Description 

_What behavior does this PR change, and why?_

This PR

* updates `kafka-clients` to 2.8.0 
* updates other dependencies to latest.

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

The demo works as expected and described in the[ relevant documentation](https://docs.confluent.io/platform/current/tutorials/examples/clients/docs/clojure.html).

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

The demo works as expected and described in the[ relevant documentation](https://docs.confluent.io/platform/current/tutorials/examples/clients/docs/clojure.html).

### CODE REVIEW

**Review by ybyzek — COMMENTED** [review]
@jsulmont thanks for the new PR.  LGTM though since IIRC @shayelkin did not give final approval on https://github.com/confluentinc/examples/pull/1025, it would be good to get the approval here before merge.

**Review by shayelkin — APPROVED** [review]
thank you! lgtm

**Review by ybyzek — APPROVED** [review]
@jsulmont thanks for the PR, @shayelkin thanks for the review! ✨

**Pull request merged**
